File: app/app/src/main/java/com/kotlin101/group2/grocerylist/data/api/models/CreateItemRequest.kt
package com.kotlin101.group2.grocerylist.data.api.models

data class CreateItemRequest(
    val alternativeItemId: Int,
    val description: String,
    val img: Any,
    val imgUrl: Any,
    val name: String,
    val pricePerUnit: Double,
    val quantity: Int
)

File: api/GroceryList/GroceryList.Api/Models/CreateItemRequest.cs
namespace GroceryList.Api.Models;

public class CreateItemRequest
{
    public string Name { get; set; }
    public string Description { get; set; }
    public decimal? PricePerUnit { get; set; }
    public int Quantity { get; set; }
    public int? AlternativeItemId { get; set; }
    public string? imgUrl { get; set; }
    public string? img { get; set; }
}
